src/HaskellCI.hs

```diff
@@ -164,25 +164,32 @@ genTravisFromConfigs argv config prj vs = do
                 , "# EOF"
                 ]
 
--- | Adjust the generated Travis YAML output with patch files, if specified.
--- We do this in a temporary file in case the user did not pass --output (as
--- it would be awkward to patch the generated output otherwise).
 patchTravis
```

Comment:
These could be moved out of

```

-------------------------------------------------------------------------------
-- Travis
-------------------------------------------------------------------------------
```

into own 

```

-------------------------------------------------------------------------------
-- Patches
-------------------------------------------------------------------------------
```

Reply:
Good idea! I've applied your suggestion.